loops.js

```diff
@@ -33,6 +40,15 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (k = 0; k < 16; k++) {
```

Comment:
De variable wordt niet gedefinieed met ' let '
Zou de variable van een for loop bij i laten beginnen dan j en dan k